Pipfile

```diff
@@ -3,6 +3,9 @@ name = "pypi"
 url = "https://pypi.org/simple"
 verify_ssl = true
 
+[packages]
+mbed-tools = {editable = true, path = "."}
```

Comment:
Why do we want mbed-tools to be editable at all?

Reply:
Some people find it useful during development as any changes are immediately picked up when you run the program. We definitely don't want it to be editable in the CI.

I don't generally find it useful to run an editable install, as there are some subtle cases where changes aren't picked up so you have to reinstall the package anyway. I didn't want to just go and remove it, and it didn't make sense to install `pytest` et al as `packages`. If you run `pipenv install -e .` it puts this in `packages` so I thought that would be the least surprising. However, I don't particularly like `pipenv` or editable installs so I went with what I thought was principle of least surprise for people who enjoy such things.